readme.md

```diff
@@ -2,7 +2,7 @@
 
 > 50+ spinners for use in the terminal
 
-![](screenshot.gif)
+[![asciicast](https://asciinema.org/a/38570.png)](https://asciinema.org/a/38570)
```

Comment:
I would still like to show the gif somewhere in the readme for the lazy people that don't bother to click.